update-dois/bin/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is only valid for Eclipse and if it contains any absolute paths, it will only be useful for *you*. This file should be removed from the repo and then ignored in .gitignore

Reply:
done